GOALS.md

```diff
@@ -0,0 +1,51 @@
+# Goals
+
+## Prescriptive
+
+1. Push Haskell towards being the language it should be, rather than the language
+    it is for historical reasons.
```

Comment:
```suggestion
Push Haskell to be a better language, this will involve removing some of the historical artifacts that are remnants of the stepping stones that got us to where we are today.
```

I think `the language it should be` is a little heavy handed, and perhaps give this sentence a tone of respect-for-our-history rather than rebellion against it.